infrastructure/lambda/template.yaml

```diff
@@ -469,6 +462,8 @@ Resources:
         Variables:
           POWERTOOLS_SERVICE_NAME: !Sub "${CriIdentifier}-checkpassport"
           ENVIRONMENT: !Ref Environment
+          DVAD_PERFORMANCE_STUB_IN_USE: !FindInMap [DVADPerformanceStubEnabledEnvVar, Environment, !Ref 'Environment']
+          DEV_ENVIRONMENT_ONLY_ENHANCED_DEBUG: !FindInMap [ DevEnvironmentOnlyEnhancedDebugMapping, Environment, !Ref Environment ]
```

Comment:
This is a great addition

Reply:
Yeh, feature flags change as parameter store vars mean reloading the lambda anyway to pick them up right away. So having them in the env vars will and changing them cause the reload as a side effect of the env var changing.